### PR TITLE
#add osm quickbuild

### PR DESCRIPTION
**Pull request by Santonia27**
Add the option in the Exposure - Asset Tab to build global FIAT model from from OSM

### CODE REVIEW

File: hydromt_fiat/api/exposure_vm.py [hunk 1]
```diff
@@ -131,6 +174,20 @@ def set_asset_locations_source(
...
+            # download OSM data
+            self.exposure_buildings_model = ExposureBuildingsSettings(
+                asset_locations=source,
+                occupancy_type=source,
+                max_potential_damage=max_potential_damage,
+                ground_floor_height=ground_floor_height,
+                unit=Units.meters.value,  # TODO: make flexible
+                extraction_method=ExtractionMethod.centroid.value,
+                damage_types=["structure", "content"],
+                damage_unit = "$",
```
Comment (panosatha) [inline hydromt_fiat/api/exposure_vm.py:187]: Is this dollars or Euros in the end?

File: hydromt_fiat/data/hydromt_fiat_catalog_global.yml [hunk 2]
```diff
@@ -27,4 +27,14 @@ jrc_damage_values:
...
+  
+country_boundaries:
+    path: country_boundaries/geoBoundariesCGAZ_ADM0.gpkg
+    data_type: GeoDataFrame
+    driver: vector
+    meta:
+      category: country boundaries
+      source_url: https://www.geoboundaries.org/globalDownloads.html
+      paper_ref: Runfola, D. et al. (2020) geoBoundaries - A global database of political administrative boundaries. PLoS ONE 15(4) - e0231866.
+      paper_doi: https://doi.org/10.1371/journal.pone.0231866
```
Comment (panosatha) [inline hydromt_fiat/data/hydromt_fiat_catalog_global.yml:40]: do we still use this?

**Review by panosatha — APPROVED** [review]
Only some minor comments. 

A general issue that I noticed is that due to the way we currently classify the OSM buildings we miss a lot of buildings that are unclassified and drop them out of the exposure. We should make an issue for that and try to find ways to correct for this.

File: hydromt_fiat/data/hydromt_fiat_catalog_global.yml (hunk 2 above)
Comment (Santonia27) [reply hydromt_fiat/data/hydromt_fiat_catalog_global.yml:40]: oh no, I forgot to remove it!

File: hydromt_fiat/api/exposure_vm.py (hunk 1 above)
Comment (Santonia27) [reply hydromt_fiat/api/exposure_vm.py]: I cannot find a conversion form Eur to USD in the code and also the damage values are based on the attache d file which is in Eur/m-2
[copy_of_global_flood_depth-damage_functions__30102017.xlsx](https://github.com/Deltares/hydromt_fiat/files/15198781/copy_of_global_flood_depth-damage_functions__30102017.xlsx)

**Pull request merged**